### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#540] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/

提示 如果你在使用过程中遇到了 bug，可以 提一个 issue 或者 加入 QQ 群 (群号: 1003437555) 进行反馈。 主题特点 ✨ 主题很大程度上继承了 VuePress 默认主题的配置，并在此基础上添加了大量功能与布局优化。

https://theme-hope.vuejs.press/zh/guide/

**Comment by LukeLiou** [comment]
在已有vue2项目里执行npm init vuepress-theme-hope add docs，vuepress dev docs后 报错
Error: Failed to resolve vue/compiler-sfc.
请问如何解决

**Reply by Mister-Hope** [reply]
VuePress 2 是基于 Vue3 的。

**Comment by firstack-it** [comment]
非常棒的主题，文档写的也很全面。

**Comment by yinuoguan** [comment]
真不错呀

**Comment by JsweetA** [comment]
请问为啥个人信息下面文章，分类，标签那块的文章没有列表了

**Reply by Mister-Hope** [reply]
文章列表现在只会显示收藏文章。

**Comment by wbvi** [comment]
您好，大佬，我是纯小白，不懂咋实现把内容加宽，侧边栏、目录向后移动。电脑端看文档，中间内容显有点挤。侧边栏、目录的两边都有空隙，若能给内容加宽，可能更方便阅读。

**Reply by Mister-Hope** [reply]
- https://github.com/vuepress-theme-hope/giscus-discussions/discussions/540
- https://github.com/
- https://weibo.com/
- https://www.baidu.com/s?wd=%E6%B5%8B%E8%AF%95
- https://www.bing.com/search?q=%E6%B5%8B%E8%AF%95
- https://mp.weixin.qq.com/s?src=11&timestamp=1700545468&ver=4909&signature=UKGXhlPxdwQydBX-8AxVGdZIRed-xVm4m2hlcwGrV5esPiTIdLns1gNowg*0UdMJsjWqP2pYgWPyAAg*BnmnGsj9OJ7e6bEfobeoeHnA05M6xqlfdbKRrtLUDHaKLiMw&new=1

Why do you think that expanding content to whole screen should be the correct option?

Or in another word, does the theme content "narrow" than them?

---

为什么你认为展开内容到整个屏幕应该是正确的选择？

或者换句话说，主题内容是否比他们“狭窄”？

---

其他文档构建器的默认样式：

- VitePress: https://vitepress.dev/guide/getting-started
- Astro: https://docs.astro.build/en/getting-started/
- Docusaurus: https://docusaurus.io/docs
- Tee Doc: https://teedoc.github.io/get_started/zh/

Only docsify is the style like you described, but it looks worst: https://docsify.js.org/#/?id=docsify

**Reply by wbvi** [reply]
内容不狭窄，浏览器缩小会显示<，可以折叠侧边栏，浏览器全屏时<消失，希望电脑访问时，侧边栏加个折叠<<，向下面的链接一样。
- https://www.feishu.cn/hc/zh-CN/articles/996038653712-%E6%AC%A2%E8%BF%8E%E6%9D%A5%E5%88%B0%E9%A3%9E%E4%B9%A6

- https://alidocs.dingtalk.com/i/p/Y7kmbokZp3pgGLq2/docs/Xzr6RBgD3LYJnadaRBkKVZPnyElvd7e9

**Reply by Mister-Hope** [reply]
https://theme-hope.vuejs.press/config/style.html#layout-config

https://theme-hope.vuejs.press/guide/interface/responsive.html

**Reply by wbvi** [reply]
好的，谢谢。

**Comment by xzy111222333** [comment]
真的，跪求各位大佬告诉我这个文档网站是用什么写的啊，好漂亮的布局

**Reply by lljl500220** [reply]
不就是vuepress吗。。漂亮只是因为hope写得漂亮

**Comment by RenderTool** [comment]
github上部署全局引入的组件是正常的，为什么码云上部署报了app-3dfcd072.js:1  Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'toLowerCase')

**Comment by caolib** [comment]
作者你好，我想要 点击文章封面也可以进入文章 该怎么做

**Reply by 00001x** [reply]
重写页面

**Comment by hopething** [comment]
怎么接入登录控制呀

**Comment by GodX-18** [comment]
666

**Comment by luohappy163** [comment]
非常好用

**Comment by luohappy163** [comment]
反对反对法

**Comment by mtueih** [comment]
有点意思
